cpp/models/abm/person.h

```diff
@@ -198,26 +198,30 @@ class Person
     }
 
     /**
-     * Every person has a random number.
-     * Depending on this number and the time, the person works from home in case of a lockdown.
+     * @brief Draw if the Person goes to work or is in home office during lockdown.
+     * Every person has a random number. Depending on this number and the time, the person works from home in case of a
+     * lockdown.
      * @return if the person works from home
      */
     bool goes_to_work(TimePoint t, const MigrationParameters& params) const;
 
     /**
+     * @brief Draw at what time the Person goes to work.
      * Every person has a random number to determine what time to go to work.
      * Depending on this number person decides what time has to go to work;
      * @return the time of going to work
```

Comment:
```suggestion
     * @return The time of going to work.
```

Also check other places.

Reply:
Left this on purpose, since I saw you changed them in #391 and it could create merge conlficts